samples/modules/paddlelite/README.rst

```diff
@@ -0,0 +1,72 @@
+.. _paddlelite:
+
+Paddle Lite sample
+###########
+
+Overview
+********
+
+This example demonstrates the Paddle Lite application's inference on a CNN model.
+The Paddle Lite version is Paddle Lite2.6. The models are trained on Baidu’s official website and cropped using the opt tool of Baidu Paddle Lite2.6.
+Due to the lack of the opencv library, a tensor whose values are all one is used to simulate an image, and the output is the output shape and corresponding probability.
+
+
+Building and Running
+********************
+
+This application can be built and executed on qemu_cortex_a53 as follows:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/modules/paddlelite
+   :host-os: unix
+   :board: qemu_cortex_a53
+   :goals: run
+   :compact:
+
+After completing the above process, you need to execute the following command to run:
```

Comment:
Why does a custom command need to be run?

Reply:
Still the same problem, we directly put the model into a space and MMU, so we need custom commands to run.